moderator/settings.py

```diff
@@ -192,8 +192,7 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
     'https://*.google-analytics.com',
     'https://*.gravatar.com',
     'https://*.wp.com',
-    'https://mozillians.org',
```

Comment:
Is this removal affecting the communication with the mozillians API?

Reply:
We only use the API through the backend.